docs/dev/run_the_etl.rst

```diff
@@ -0,0 +1,88 @@
+.. _run-the-etl:
+
+=======================
+Walk, Don't Run the ETL
+=======================
+
+So you want to run the PUDL data processing pipeline? This is the most involved
+way to get access to PUDL data. It is only suggested if you want to edit the
+ETL process or contribute to the code base. Check the basic usage page for the
+other access methods (see: :ref:`basic-usage`). Measure twice, cut once.
+
+These instructions assume you have already gone through the development setup
+(see: :ref:`dev_setup`).
+
+There are four main scripts that are involved in the PUDL processing pipeline:
+
+1. `ferc1_to_sqlite` :doc: clones FERC Form 1's dbf files <clone_ferc1> as
+   a sqlite database - a necessary prep extract step for FERC Form 1.
+2. `pudl_etl` is where the magic happens. This is the main script which
```

Comment:
There's now also `censusdp1tract_to_sqlite` if you want to add that in here -- required for any of the geospatial stuff.

Reply:
I guess the `censusdp1_to_sqlite` conversion will get called if someone tries to access the DP1 SQLite DB and it's not there yet, but that step does have to happen somehow.